### DISCUSSION TITLE
Upgrade from 5.6.4 to 6.2.0 fails

### DISCUSSION THREAD

**Discussion by elliotn** [ampache/ampache#3743] [Q&A] [answered]
I'm trying and failing to upgrade. I switch branches from "release5" to "release6". I have php v8.2.14 installed. Unfortunately, the database update fails with a stack trace. Any suggestions would be appreciated.

\# php bin/cli admin:updateDatabase -e

Ampache version: 6.2.0
Config version: 69
Database version: 6.0.0 Build: 004

Update Now!

_update_600005
Error Typed property Ampache\Repository\Model\Album::$version must not be accessed before initialization
(thrown in /var/www/html/ampache5.via.git/src/Repository/Model/Album.php:526)

Stack Trace:

  0) Ampache\Repository\Model\Album->get_fullname()
     at /var/www/html/ampache5.via.git/src/Repository/Model/Album.php:157
  1) Ampache\Repository\Model\Album->__construct()
     at /var/www/html/ampache5.via.git/src/Module/System/Update.php:4509
  2) Ampache\Module\System\Update::_update_600005()  3) call_user_func()
     at /var/www/html/ampache5.via.git/src/Module/System/Update.php:1002
  4) Ampache\Module\System\Update::run_update()
     at /var/www/html/ampache5.via.git/src/Module/Cli/AdminUpdateDatabaseCommand.php:103
  5) Ampache\Module\Cli\AdminUpdateDatabaseCommand->execute()
     at /var/www/html/ampache5.via.git/vendor/adhocore/cli/src/Application.php:366
  6) Ahc\Cli\Application->doAction()
     at /var/www/html/ampache5.via.git/vendor/adhocore/cli/src/Application.php:280
  7) Ahc\Cli\Application->handle()
     at /var/www/html/ampache5.via.git/bin/cli:82

**Comment by elliotn** [comment]
Never mind. I gave up on doing an in-place upgrade. I've creating a new installation for release 6 and trying to copy the various settings that I had.

**Comment by lachlan-00** [comment, marked as answer]
the change in a later db update wouldn't work unless you were at a certail level so i've fixed that with https://github.com/ampache/ampache/issues/3748#issuecomment-1873619118 and i'll do a release this week to update it
